[ENG-895] feat: update Slack App documentation (#59)
currents-dev/currents-readme: ai/overview.md

| [MCP Server](mcp-server.md)                                  | Any MCP-capable agent             | Agents querying runs, tests, and analytics on demand; autonomous troubleshooting |
| [IDE Extension](ide-extension.md)                            | VS Code, Cursor, compatible forks | Debugging CI failures without leaving the editor                                 |
| [Fix with AI](overview.md#fix-with-ai-from-the-dashboard)    | Currents dashboard                | Handing a failure to an agent while triaging a run                               |

> | [Slack Fix with AI](../resources/integrations/slack/slack-app.md#fix-with-ai) | Slack | Handing a failed-test notification to an agent from a team channel |

Reviewer: ```suggestion
| [Slack Fix with AI](../resources/integrations/slack/slack-app.md#fix-with-ai) | Slack | Handing a failed-test notification to an agent from a channel |
```

This is the bridge from investigation to action: whoever is looking at the failing run - not necessarily the person with the repo open - can package the failure with its full context and route it to an agent in one click.

### Fix with AI from Slack

> Failed-test notifications from the [Slack App](../resources/integrations/slack/slack-app.md#fix-with-ai) include a **Fix with AI** button. The modal can open the failure in Cursor or GitHub Copilot, provide a copyable prompt for Claude, Codex, Zed, Conductor, and other tools, and link to the Currents MCP installation. This lets anyone monitoring a team channel route a failure to an agent without first opening the Currents dashboard.

Reviewer: Can we rephrase this, we start with "The modal", the user is unaware that a model will open.

> Failed-test notifications from the [Slack App](../resources/integrations/slack/slack-app.md#fix-with-ai) include a **Fix with AI** button. Clicking it opens options to route the failure to Cursor or GitHub Copilot, copy a prompt for Claude, Codex, Zed, Conductor, and other tools, or open the Currents MCP installation. This lets anyone monitoring a team channel route a failure to an agent without first opening the Currents dashboard.

Reviewer: ```suggestion
Failed-test notifications from the [Slack App](../resources/integrations/slack/slack-app.md#fix-with-ai) include a **Fix with AI** button. This sends failed-test context to Cursor, GitHub Copilot, Claude, Codex, Zed, Conductor, and other AI agents. Anyone monitoring a team channel can route a failure to an agent without opening the Currents dashboard.
```
